.nuget/create_nuget.py

```diff
@@ -25,11 +25,11 @@
     copy_file(os.path.join(source, "lib_lightgbm.dll"), os.path.join(windows_folder_path, "lib_lightgbm.dll"))
     copy_file(os.path.join(source, "lightgbm.exe"), os.path.join(windows_folder_path, "lightgbm.exe"))
     version = open(os.path.join(current_dir, os.path.pardir, 'VERSION.txt')).read().strip().replace('rc', '-rc')
-    nuget_str = r"""<?xml version="1.0"?>
+    nuget_str = f"""<?xml version="1.0"?>
     <package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
     <metadata>
         <id>LightGBM</id>
-        <version>%s</version>
+        <version>{version, datetime.datetime.now().year}</version>
```

Comment:
This fix is not quite correct. The `%s` should be replaced with `version` and the `%d` on line 39 should be replaced with `datetime.datetime.now().year`.

Reply:
fixed it